webapi/src/main/scala/org/knora/webapi/messages/admin/responder/listsmessages/ListsMessagesADM.scala

```diff
@@ -180,17 +199,24 @@ final case class NodeADM(nodeinfo: ListChildNodeInfoADM, children: Seq[ListChild
    * @return a sorted [[List]].
    */
   def sorted: NodeADM = this.copy(children = children.sortBy(_.position).map(_.sorted))
+
+  def withChildren: Seq[ListNode] = nodeinfo +: children
 }
 object NodeADM {
   implicit lazy val codec: JsonCodec[NodeADM] = DeriveJsonCodec.gen[NodeADM]
   implicit def schema: Schema[NodeADM]        = Schema.derived[NodeADM]
 }
 
+sealed trait ListNode {
+  def id: IRI
+  def labels: StringLiteralSequenceV2
```

Comment:
Nice.

I like this addition, I would have probably also removed the definitions from the sealed traits extending `ListNode` and move the scaladoc here.